quipucords/tests/quipucords/test_user.py

```diff
@@ -0,0 +1,65 @@
+"""Test user module."""
+
+import re
+
+import pytest
+from django.contrib.auth import get_user_model, password_validation
+from django.core.exceptions import ValidationError
+
+from quipucords import settings
+from quipucords.user import create_random_password
+
+User = get_user_model()
+
+
+class TestUserRandomPasswords:
+    """Tests to verify user module functions."""
+
+    def test_create_random_password_size(self):
+        """Test user random password is minimum size."""
+        assert len(create_random_password()) == settings.QPC_MINIMUM_PASSWORD_LENGTH
+
+    def test_create_random_password_includes_allowed_characters(self):
+        """Test user random password includes only allowed characters."""
+        random_password = create_random_password()
+        assert re.search("[a-zA-Z0-9]*", random_password).group() == random_password
+
+
+class TestUserPasswordValidators:
+    """Tests the user password validators that we imposed."""
+
+    def test_minimum_size(self):
+        """Test that we impose a minimum size."""
+        with pytest.raises(ValidationError, match="This password is too short"):
+            password_validation.validate_password("hjK4")
+
+    def test_reject_common_words(self):
+        """Test that we reject common words."""
+        with pytest.raises(ValidationError, match="This password is too common"):
+            password_validation.validate_password("hello")
+
+    def test_reject_qpc_common_words_qpc(self):
+        """Test that we reject quipucords common words."""
+        with pytest.raises(ValidationError, match="This password is too common"):
+            password_validation.validate_password("qpcpassw0rd")
+
+    def test_reject_qpc_common_words_dsc(self):
+        """Test that we reject quipucords common words."""
+        with pytest.raises(ValidationError, match="This password is too common"):
+            password_validation.validate_password("dscpassw0rd")
+
```

Comment:
this part can be DRYfied with parametrization =D

```suggestion
@pytest.mark.parametrize("password", ["hello", "qpcpassw0rd", "dscpassw0rd"])
    def test_reject_common_words(self, password):
        """Test that we reject common words."""
        with pytest.raises(ValidationError, match="This password is too common"):
            password_validation.validate_password(password)

```